etc/HIReplayOfflineConfiguration.py

```diff
@@ -89,33 +106,38 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-       'default': "CMSSW_10_3_1_patch2"
-     }
+    'default': "CMSSW_12_5_2_patch1"
+}
 
 # Configure ScramArch
-setDefaultScramArch(tier0Config, "slc6_amd64_gcc700")
+setDefaultScramArch(tier0Config, "el8_amd64_gcc10")
 
 # Configure scenarios
-ppScenario = "ppEra_Run2_2018"
-ppScenarioB0T = "ppEra_Run2_2018"
-cosmicsScenario = "cosmicsEra_Run2_2018"
-hcalnzsScenario="hcalnzsEra_Run2_2018_pp_on_AA"
-hiScenario = "ppEra_Run2_2016_pA"
-alcaTrackingOnlyScenario = "trackingOnlyEra_Run2_2018"
-HIalcaTrackingOnlyScenario = "trackingOnlyEra_Run2_2018_pp_on_AA"
+ppScenario = "ppEra_Run3"
+ppScenarioB0T = "ppEra_Run3"
+cosmicsScenario = "cosmicsEra_Run3"
+hcalnzsScenario = "hcalnzsEra_Run3"
+HIhcalnzsScenario = "hcalnzsEra_Run3_pp_on_PbPb"
+hiScenario = "ppEra_Run3_pp_on_PbPb"
+alcaTrackingOnlyScenario = "trackingOnlyEra_Run3"
+HIalcaTrackingOnlyScenario = "trackingOnlyEra_Run3_pp_on_PbPb"
 alcaTestEnableScenario = "AlCaTestEnable"
-alcaLumiPixelsScenario = "AlCaLumiPixels"
-hiTestppScenario = "ppEra_Run2_2018_pp_on_AA"
-
-# Defaults for processing version
-defaultProcVersion = 221
-expressProcVersion = 221
-alcarawProcVersion = 221
+alcaLumiPixelsScenario = "AlCaLumiPixels_Run3"
+alcaPPSScenario = "AlCaPPS_Run3"
+hiTestppScenario = "ppEra_Run3_pp_on_PbPb"
+hiRawPrimeScenario = "ppEra_Run3_pp_on_PbPb_approxSiStripClusters"
+
+# Procesing version number replays
+# Taking Replay processing ID from the last 8 digits of the DeploymentID
+dt = int(open("DeploymentID.txt","r").read()[4:])
+defaultProcVersion = dt
+expressProcVersion = dt
+alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "103X_dataRun2_Express_v2"
-promptrecoGlobalTag = "103X_dataRun2_Prompt_v3"
-alcap0GlobalTag = "103X_dataRun2_Prompt_v3"
+expressGlobalTag = "124X_dataRun3_Express_HIreplay2022_v1"
+promptrecoGlobalTag = "124X_dataRun3_Prompt_HIreplay2022_v1"
+alcap0GlobalTag = "124X_dataRun3_Prompt_HIreplay2022_v1"
```

Comment:
@germanfgv please use the "prod" GTs here as well:
```suggestion
expressGlobalTag = "124X_dataRun3_Express_v9"
promptrecoGlobalTag = "124X_dataRun3_Prompt_v10"
alcap0GlobalTag = "124X_dataRun3_Prompt_v10"
```

Reply:
Done